packages/thirdweb/src/wallets/smart/lib/userop.ts

```diff
@@ -191,7 +419,10 @@ export async function createUnsignedUserOp(args: {
       entrypointAddress: overrides?.entrypointAddress,
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +415]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Could we call this `paymasterEstimateResult` or something like that? the result and result2 names are really hard to parse at first<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4106?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->